xml/Microsoft.VisualBasic.Vsa/VsaEngine.xml

```diff
@@ -431,7 +432,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a value that indicates whether the current in-memory representation of the item differs from the persisted representation.</summary>
```

Comment:
```suggestion
        <summary>Gets a value that indicates whether the script engine's source state has changed since the last save operation, or since the script engine was originally loaded.</summary>
```